website/sidebars.ts

```diff
@@ -61,6 +61,7 @@ const sidebars: SidebarsConfig = {
       items: [
         'node-operators/alpha-testing',
         'node-operators/webnode/local-webnode',
+        'node-operators/webnode/local-webnode-docker',
```

Comment:
I would be in favor of having this part moved to `Node Operations` or something else soon. It can be done in a follow-up. Can you open an issue, please?